src/commands/RunCommand.js

```diff
@@ -42,8 +42,7 @@ export default class RunCommand extends Command {
   }
 
   runScriptInPackage(pkg, callback) {
-    NpmUtilities.runScriptInDir(this.script, this.args, pkg.location, (err, stdout) => {
-      this.logger.info(stdout);
+    NpmUtilities.runScriptInDir(this.script, this.args, pkg.location, (err) => {
       if (err) {
```

Comment:
Oh, I mean why not log it here _instead_ of in `exec`?

Do we always want `exec` to log stdout on error?